examples/abstract-objects/abstract-objects.description

```diff
@@ -1,3 +1,3 @@
-// In Ballerina, objects can be abstract. These objects only describes the type of
-// each field and method. These objects cannot be initialized, and does not have a
-// default value.
+// In Ballerina, objects can be abstract. Abstract objects only describes the type of
+// each field and method. An abstract object cannot be initialized, and does not have 
```

Comment:
Minor correction:
"// In Ballerina, objects can be abstract. Abstract objects only describe 
// each field type and method. An abstract object cannot be initialized, and does not have"